packages/passport/sdk-sample-app/src/context/ImmutableProvider.tsx

```diff
@@ -90,12 +102,7 @@ const getPassportConfig = (environment: EnvironmentNames): PassportModuleConfigu
           imxApiClients: new ImxApiClients(createConfig({
             basePath: 'https://api.dev.immutable.com',
           })),
-          immutableXClient: new IMXClient({
-            baseConfig,
-            overrides: {
-              immutableXConfig: getSdkConfig(EnvironmentNames.DEV),
-            },
-          }),
+          immutableXClient: new IMXClient(getSdkConfig(EnvironmentNames.DEV)),
```

Comment:
this change removed baseConfig content from the xclient value. is this intentional?

Reply:
yes, because now the getSdkConfig function includes the baseConfig (it didn't previously)

Reply:
(and the overrides)